src/utils.js

```diff
@@ -239,6 +239,10 @@ export const getNextSelectedRowsState = (selectedRows, row) => {
   return newSelectedRows;
 };
 
+export const extractPickSlipRequestIds = (pickSlipsData) => {
+  return pickSlipsData.map(pickSlip => pickSlip?.request?.requestID);
+};
+
```

Comment:
I think this can be either -
more generalized to accept array and path
or 
place it in the module where you are using it, probably outside or inside the component